### PR TITLE
Add missing step about adding to GitHub's Releases section

### PR DESCRIPTION
**Pull request by Purg**

### CODE REVIEW

**Review by brianhhu — COMMENTED** [review]
Please take a look at my comment about whether the repetition in draft release directions is intentional or not. Thanks!

**Review by brianhhu — APPROVED** [review]
LGTM, thanks- there is an extra link here that may no longer be needed.

File: docs/release_process.rst [hunk 1]
```diff
@@ -159,22 +163,34 @@ current package version.
...
+e. Copy and paste the release notes for this version into the description
+   field.
+
+f. Select the "This is a pre-release" check-box if applicable.
+
+g. Click the "Publish Release" button to create the GitHub release!
 
 
 .. _Poetry's version command: https://python-poetry.org/docs/cli/#version
 .. _releases page on GitHub: https://github.com/XAITK/xaitk-saliency/releases
```
[the remark this answers is not in the document]
Comment (Purg) [reply docs/release_process.rst:194]: Removed.

**Review by brianhhu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**